pontoon/sync/vcs/models.py

```diff
@@ -151,36 +151,22 @@ def find_changed_files(repo, locale=None):
                     for path in find_changed_files(repo, locale):
                         files.setdefault(path, []).append(locale)
             else:
+                locale_directories = self.locale_directories(repo)
+                locale_directories_paths = locale_directories.keys()
+
                 for changed_file in find_changed_files(repo):
-                    path_info = self.get_path_info(changed_file, repo)
+                    if is_hidden(changed_file):
+                        continue
 
-                    if path_info:
-                        path, locale_path, locale = path_info
-                        path = path[len(locale_path):].lstrip(os.sep)
-                        files.setdefault(path, []).append(locale)
+                    for ldp in locale_directories_paths:
```

Comment:
Erm, right... Good idea.  `get_path_info` is a legacy from the one of the more "functional" versions of this code.9